remote/remote.go

```diff
@@ -740,8 +731,13 @@ func runWpCliCmdRemote(conn net.Conn, GUID string, rows uint16, cols uint16, wpC
 		for {
 			select {
 			case <-ticker:
-				if (!wpcli.Running && wpcli.BytesStreamed[remoteAddress] >= wpcli.BytesLogged) || nil == conn {
-					log.Println("WP CLI command finished and all data has been written, exiting this watcher loop")
+				if nil == conn {
+					log.Println("client connection is closed, exiting this watcher loop" )
+					break Exit_Loop
+				}
+
+				if (!wpcli.Running && wpcli.BytesStreamed[remoteAddress] >= wpcli.BytesLogged) {
+					log.Println("WP CLI command finished and all data has been written, exiting this watcher loop 123")
```

Comment:
@luismulinari is the 123 intentional? Because on one side I'd love to be able to differentiate the logs from the two similar loops. They don't help understanding what's happening when we debug